pylhe/__init__.py

```diff
@@ -133,8 +134,8 @@ def readLHEInit(thefile):
                 if child.tag == "weightgroup" and child.attrib != {}:
                     try:
                         wg_type = child.attrib["type"]
-                    except Exception, e:
-                        print "weightgroup must have attribute 'type'"
+                    except Exception:
+                        print("weightgroup must have attribute 'type'")
                         sys.exit(1)
```

Comment:
can we just re-raise here instead of exiting?

Reply:
probably should be KeyError

Reply:
Yes, definitely. These are just the things that were in before from the other PR (with a quick edit to get pyflakes passing).

Reply:
Updated.